### ISSUE TITLE
Multi-Column Layout of Completion Candidates

### ISSUE THREAD

**Issue by nordlow** [abo-abo/swiper#2846]
I'm missing the option to display completion candidates in multi-column layout like Vanilla Emacs does. Is this planned?

**Comment by basil-conto** [comment]
> Is this planned?

No, and I'd say the single-column layout is pretty firmly entrenched in the current design.  But don't let that stop you (or anyone else) from trying ;).

Perhaps the first thing to try would be a custom entry in `ivy-format-functions-alist`, or `ivy-display-functions-alist` and `ivy-display-functions-props`.
